content/gettingstarted/index.adoc

```diff
@@ -39,88 +39,67 @@ icon:lightbulb[size=1x]{nbsp}For several commands listed on this page, the OpenS
-
-[source,bash]
-----
-minikube addons enable ingress
-----
-
-When Kiali is installed in Minikube, this Ingress add-on will let you access the Kiali UI at the URL `https://[minikube-ip]/kiali/console` where `[minikube-ip]` is what is reported when you run the command: `minikube ip`.
-
 === Install Istio
 
-Make sure your cluster environment has Istio installed. Because you are following these instructions to install the latest version of Kiali, you should not install the Kiali version that is bundled with Istio (that is, do not use the Helm setting `kiali.enabled=true` when installing Istio).
-
+Make sure your cluster environment has Istio installed.
 For details on how to install Istio, see the link:https://istio.io/docs/setup/[Istio Setup docs].
 
-=== Kiali Operator
-
-Now install the latest release of Kiali using the Kiali Operator.
-
-==== What is the Kiali Operator?
-
-First, a quick introduction of what the Kiali Operator actually is.
+=== Install the Kiali Operator
 
 The Kiali Operator is an implementation of a link:https://coreos.com/operators/[Kubernetes Operator]. It provides a method of packaging, deploying, and managing Kiali.
 
-The Kiali Operator runs inside the `kiali-operator` namespace and listens for the creation of a Kiali CR (custom resource) object. When it sees one created, it will install Kiali using the settings it finds in the CR object. To see all the different settings you can configure in a Kiali CR, see link:https://github.com/kiali/kiali/blob/master/operator/deploy/kiali/kiali_cr.yaml[this fully documented example Kiali CR yaml file]. A minimal Kiali CR that uses all the defaults looks like this:
+To install the Kiali Operator (and, by default, the Kiali CR), run the install script via this command:
 
-[source,yaml]
+[source,bash]
 ----
-apiVersion: kiali.io/v1alpha1
-kind: Kiali
-metadata:
-  name: kiali
+bash <(curl -L https://git.io/getLatestKialiOperator)
 ----
 
-==== Installing the Kiali Operator
-
-In the instructions below, you will install the Kiali Operator using the link:https://github.com/kiali/kiali/blob/master/operator/deploy/deploy-kiali-operator.sh[Kiali Operator install script]. That install script has the option of also immediately installing a simple Kiali CR for you so you do not have to, thus making it easy to get started with Kiali by running a single command (this is, in fact, the install script's default behavior). If, however, you wish to install Kiali specifically configured with custom settings of your choosing, you can tell the script to skip the installation of a Kiali CR and just simply install the Operator. In this case, you would be responsible for writing the Kiali CR yaml file and creating it in your cluster, thus allowing you to tell the Operator to install Kiali with the settings you want.
+It will pull the latest version of Kiali Operator and Kiali itself, and install them with default options. If Kiali was already installed, you will be prompted about removing it first.
```

Comment:
```suggestion
This will pull the latest version of the Kiali Operator and Kiali itself, and install them using the default options. If Kiali was already installed, you will be prompted to remove the older version.
```